src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/ProcessController.cs

```diff
@@ -381,22 +386,31 @@ private async Task<bool> CanTaskBeEnded(Instance instance, string currentElement
 
                 string nextElement = nextElements.First();
 
-                ProcessStateChange nextResult =  _processService.ProcessNext(instance, nextElement, User);
-
-                if (nextResult != null)
+                try
                 {
```

Comment:
Are we missing authorization here ?

Reply:
well spotted. I've included some lines which will be in next nuget.